examples/README.md

```diff
@@ -1,7 +1,10 @@
 # Examples
 
 These examples demonstrate the main features of Bevy and how to use them.
```

Comment:
I think I would prefer to leave non-default feature documentation out of the README. Instead, lets create a new `docs/features.md` file and link to it from the readme.